app/helpers/otus/catalog_helper.rb

```diff
@@ -101,9 +120,12 @@ def otu_descendants_and_synonyms(
         end
         data[:leaf_node] = data[:descendants].empty?
       else
-        data[:leaf_node] = descendants.where.not(id: similar_otus).none?
```

Comment:
see about adding `.unscope(:order)` on call using `descendant` etc. it can save a lot of time`